scalafmt-tests/src/test/resources/scala3/OpaqueType.stat

```diff
@@ -15,8 +15,15 @@ maxColumn = 15
 ===
 opaque type VeryLongClassName <: A & B = AB
 >>>
-opaque type VeryLongClassName <: A & B =
```

Comment:
So this started do have A and B on separate lines due to the recent fixes and I thought it might be better to split on `<:`

Reply:
definitely better than on `&`. doesn't look like you did anything to penalize breaking on `&`, though, right?

Reply:
No, would it work when we wrap or/and with singleLine policy then? Did you mean this in the other comment?

Reply:
see comment above.

Reply:
Changed the test here and added some into `UnionIntersectionType.stat`